platform/android/java/app/config.gradle

```diff
@@ -22,3 +24,27 @@ ext.getExportPackageName = { ->
 	}
 	return appId
 }
+
+/**
+ * Parse the project properties for the 'custom_template_v2_plugins' property and return
+ * their binaries for inclusion in the build dependencies.
+ *
+ * The listed plugins must have their binaries in the 'libs/plugins' directory.
+ */
+ext.getGodotPluginsBinaries = { ->
+	// Default dependencies
```

Comment:
Also mixed tabs and spaces. (Including in some preexisting code). I think settling on 4-spaces would be good, unless there's a reason to prefer tabs, but it should be consistent in any case.